tests/moe/test_w4a8_mxfp4_moe.py

```
@@ -0,0 +1,165 @@
+intermediate precisions (activation -> FP8, SwiGLU output -> FP8) so the check isolates
+kernel correctness from quantization error.
+"""
+
+import pytest
+import torch
+import torch.nn.functional as F
+
+try:
+    try:
+        from flashinfer.fused_moe import w4a8_mxfp4_moe
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)